packages/server/graphql/mutations/createJiraTaskIntegration.ts

```diff
@@ -69,15 +72,55 @@ export default {
       )
     }
 
-    const viewerAuth = await dataLoader.get('freshAtlassianAuth').load({teamId, userId: viewerId})
-    if (!viewerAuth) {
-      return standardError(new Error('The assignee does not have access to Jira'), {
+    const [viewerAuth, assigneeAuth, team] = await Promise.all([
+      dataLoader.get('freshAtlassianAuth').load({teamId, userId: viewerId}),
+      dataLoader.get('freshAtlassianAuth').load({teamId, userId}),
```

Comment:
let's remove 
```
    if (!userId) {
      // we should probably remove this constraint in the future
      return {error: {message: 'Task must have assignee before it can be integrated with Jira'}}
    }
```

and then we can do the same checks here that you do for the github version:
- check userId exists before calling the dataloader
- check userId exists before createdBySomeoneElseComment